<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearPublicacion</class>
 <widget class="QMainWindow" name="CrearPublicacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 229, 158);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCrear">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CREAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCorreo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>HORA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>FECHA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCorreo_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>CONTEIDO:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImagen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>AGREGAR IMAGEN</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtContenido">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>691</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="txtFecha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>IMAGEN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtImagen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 20pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTimeEdit" name="txtHora">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>435</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
